guides/V1_Meeting.md

```diff
@@ -0,0 +1,79 @@
+---
+copyright: 'Copyright IBM Corp. 2018'
+link: 'meeting'
+is: 'experimental'
+---
+
+# Get meeting details
+
+## Concepts
+
+The _meeting_ query allows the API caller to get details about the meeting.  The query accepts a Space ID (spaceId) and will return meeting details if the calling user has access to the meeting.  The calling user will have access to the meeting if the user has access to the space which contains the meeting, or the meeting has been marked to allow public participants.
+
+## Schema
+
+### Meeting Query
+
+
+```graphql
+type QueryRoot {
+  ...
+  meeting(spaceId: ID!): Meeting
+}
+
+type Meeting {
+  active: Boolean!
+  meetingNumber: String
+  password: String
+  allowPublic: Boolean!
+  startTime: Date
+  joinInfo: JoinInfo
+}
+
+type JoinInfo {
+  meetingProviderJoinUrl: String
+  pstnPasscode: String
+  dialInNumbers : [DialInNumber]
+}
+
+type DialInNumber {
+  countryCode: String!
+  number: String!
+}
+
+```
+
+## Example Request
+
+~~~~
+Method: POST
```

Comment:
In the other guides we use a link to the GraphQL tool to show queries like this - so you can just copy/paste the query into the tool, then grab the URL from the tool. You will need apiType=experimental on the URL for this one. I was recently doing the moments doc in a similar way if you want to check it out.